cmd/kubelet/app/server.go

```diff
@@ -717,8 +717,6 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		}
 	}
 
-	// Setup event recorder if required.
-	makeEventRecorder(kubeDeps, nodeName)
 
```

Comment:
delete this blank line to pass golangci-lint

Reply:
ok! Thanks for reminding